data/json/items/tool/cooking.json

```diff
@@ -572,15 +572,15 @@
     "description": "A self-contained camping mess kit, containing everything you might need for wilderness cooking.",
     "weight": "500 g",
     "volume": "1100 ml",
-    "longest_side": "12 cm",
+    "longest_side": "15 cm",
     "price": 3000,
     "price_postapoc": 100,
     "to_hit": -1,
     "material": [ "aluminum" ],
     "symbol": ";",
     "color": "green",
-    "qualities": [ [ "BOIL", 2 ], [ "CONTAIN", 1 ] ],
-    "pocket_data": [ { "max_contains_volume": "500 ml", "max_contains_weight": "2 kg", "watertight": true, "rigid": true } ],
+    "qualities": [ [ "COOK", 3 ], [ "BOIL", 2 ], [ "CONTAIN", 1 ], [ "CHEM", 1 ] ],
+    "pocket_data": [ { "max_contains_volume": "1000 ml", "max_contains_weight": "2 kg", "watertight": true, "rigid": true } ],
```

Comment:
I was basing off this https://www.amazon.com/gp/product/B01743BX1A/ref=as_li_ss_tl?ie=UTF8&linkCode=sl1&language=en_US&th=1

But you should know best what the correct volume/liter is

Reply:
This would put the item at roughly 2l (somewhere around 2050-2100ml) of volume and increase the weight roughly 20 grams. If you want to apply these changes be my guest. Edits aren't bad, they just need to be founded on something and with a link you have a foundation for yours.

All I'd ask is that if you do end up correcting the values please do me a favor and include a json note with the link for this amazon listing, the same way I do to items I edit nowadays. It will make it far easier to tell that the numbers aren't arbitrary.

Reply:
Yep, it looks like you've done it right. I don't believe there's a way to have *working* links in JSON, but one can just copy it into their browser and it'll work well enough. Thank you for doing this!